lib/redshift_connector/importer/rebuild_rename.rb

```diff
@@ -14,12 +14,11 @@ def execute(bundle)
       tmp_table = "#{dest_table}_new"
       old_table = "#{dest_table}_old"
 
-      tmp_dao = @dao.dup
-      tmp_dao.table_name = tmp_table
+      @dao.table_name = tmp_table
```

Comment:
This is destructive operation, naming tmp_dao such like the following code is better.
```
tmp_dao = @dao.dup
self.class.const_set(:TmpDAO_00001, tmp_dao)   # give the tmp_dao the const path
tmp_dao.name   # force tmp_dao to have the class name
```

Reply:
👍  fixed: https://github.com/bricolages/redshift-connector/pull/13/commits/dfcf1b1e871f32172c40ec04d71d562a7318545d